src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/formViewNetworkWideValidationSchema.tsx

```diff
@@ -89,15 +93,27 @@ export const networkWideValidationSchema = Yup.lazy<FormViewNetworkWideValues>(
       vlanId: Yup.mixed().when('useVlan', {
         is: true,
         then: Yup.number()
-          .required(REQUIRED_MESSAGE)
+          .required(MUST_BE_A_NUMBER)
           .min(1, `Must be more than or equal to 1`)
           .max(MAX_VLAN_ID, `Must be less than or equal to ${MAX_VLAN_ID}`)
+          .test('not-number', MUST_BE_A_NUMBER, (value) =>
+            validateNumber(value as number, values.vlanId),
```

Comment:
Why not only use `values.vlanId`? Then we'd be validating exactly what the user entered.